app/ui/navbar.tsx

```diff
@@ -13,20 +13,51 @@ import Avatar from '@mui/material/Avatar';
 import Button from '@mui/material/Button';
 import Tooltip from '@mui/material/Tooltip';
 import MenuItem from '@mui/material/MenuItem';
-import Image from "next/image";
+import Image from 'next/image';
 import Link from '@mui/material/Link';
+import { TextField } from '@mui/material';
+import { Stack } from '@mui/material';
+import { Autocomplete } from '@mui/material';
+import { useState, useEffect } from 'react';
+import { Movie } from '../lib/definitions';
 
 const links = [
-  { name: 'Om oss', href: '/om-oss'},
-  { name: 'Aktuella visningar', href: '/aktuella-vinsningar'},
-  { name: 'Filmer', href: '/filmer'},
-  { name: 'Biljetter', href: '/biljetter'},
+  { name: 'Om oss', href: '/om-oss' },
+  { name: 'Aktuella visningar', href: '/aktuella-vinsningar' },
+  { name: 'Filmer', href: '/filmer' },
+  { name: 'Biljetter', href: '/biljetter' },
 ];
 const settings = ['Mina biljetter', 'inställningar', 'Logout'];
 
 export default function ResponsiveAppBar() {
   const [anchorElNav, setAnchorElNav] = React.useState<null | HTMLElement>(null);
   const [anchorElUser, setAnchorElUser] = React.useState<null | HTMLElement>(null);
+  const [movies, setMovies] = useState<Movie[]>([]);
+  const [filteredMovies, setFilteredMovies] = useState<Movie[]>([]);
+  const [isFocused, setIsFocused] = useState(false);
+
+  useEffect(() => {
+    const fetchMovies = async () => {
+      try {
+        const response = await fetch('/api/movies');
+        const data = await response.json();
+        if (Array.isArray(data.movies)) {
+          const movieData = data.movies.map((movie: any) => ({
+            title: movie.title,
+            id: movie._id,
+            link: `/movies/${movie._id}`,
```

Comment:
Here is one place where it should be filmer instead of movies

Reply:
that's actually dead code, so thanks for telling me!